src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -450,9 +420,7 @@ protected async Task BuildAndRunTest(string appName, Project project, BrowserKin
         private async Task TestBasicNavigation(string appName, IPage page, bool usesAuth = false, bool skipFetchData = false)
         {
             await page.WaitForSelectorAsync("nav");
-
-            // <title> element gets project ID injected into it during template execution
-            Assert.Equal(appName.Trim(), (await page.GetTitleAsync()).Trim());
+            Assert.Equal(usesAuth ? appName.Trim() : "Index", (await page.GetTitleAsync()).Trim());
```

Comment:
This seems wrong. Why would the page title depend on whether the project uses auth? Is the template with auth not correctly updated to include the `<PageTitle>` components?

Reply:
There might be a race condition here between the title by default and what gets applied by the head component (I think). 

I'm trying to get the test to just pass since it was broken before and its not relevant to this update (and we don't have to be functionally testing the head component on the templates)

I'm happy to file an issue if we want to follow up on this, however I don't want this PR to become let's fix these tests.

Reply:
Totally understand not wanting to expand the scope of the PR to include fixing unrelated tests.

However it is dangerous to have assertions that we don't understand, especially relating to possible race conditions, given how sensitive people are about their view that browser automation tests are flaky.

I would rather just remove this assertion entirely than have it be here doing something we don't understand. If you think the assertion is necessary in order for the test to say something worthwhile, then yes please also file an issue saying that someone should investigate what's needed to put back an assertion about the page title.

Reply:
I agree, however we need to have a way to determine we are ready to continue testing (which is what we used this assertion for)

Reply:
Filed an issue to improve the mechanism https://github.com/dotnet/aspnetcore/issues/36224 will tackle it on a follow-up

Reply:
Let's not pick something that's subject to a race condition :)

Reply:
> I agree, however we need to have a way to determine we are ready to continue testing

Also, are you sure that's true? It's using `Assert.Equal`, not `Browser.Equal`, so there's nothing "waiting" here, unless I'm misreading it. Am I misunderstanding?

Reply:
I've fixed this by moving the assert below the one that checks the index page is rendered, that should get rid of the race condition, since at that time the title should already be set

Reply:
Awesome, great solution